search/includes/classes/commands/class-documentcommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Automattic\VIP\Search\Commands;
+
+use \WP_CLI;
+use \WP_CLI\Utils;
+
+/**
+ * Commands to view and manage individual documents
+ *
+ * @package Automattic\VIP\Search
+ */
+class DocumentCommand extends \WPCOM_VIP_CLI_Command {
+	private const SUCCESS_ICON = "\u{2705}"; // unicode check mark
+	private const FAILURE_ICON = "\u{274C}"; // unicode cross mark
+
+
+	/**
+	 * Get details about a document by type and an id
+	 *
+	 * ## OPTIONS
+	 *
+	 * <type>
+	 * : The index type (the slug of the Indexable, such as 'post', 'user', etc)
+	 * 
+	 * <object_id>
+	 * : The ID of the object
+	 * 
+	 * ## EXAMPLES
+	 *     wp vip-search documents get post 2
+	 *
+	 * @subcommand get
+	 */
+	public function get( $args, $assoc_args ) {
+		$type = $args[0];
+		$object_id = $args[1];
+	
+		$search = \Automattic\VIP\Search\Search::instance();
```

Comment:
Should this logic be placed in some business logic class, or is it ok to keep here?

Is there a way to unit-test this?

Reply:
It looks okay to me. 

Unit testing WPCLI commands is difficult. If the individual components are unit tested, it's fine for now.